todo_app/Tests/test_app.py

```diff
@@ -0,0 +1,192 @@
+import os, pytest, json, datetime
+import todo_app.app
+import dotenv
+from unittest.mock import patch, Mock
+
+trello_board_id = os.getenv('TRELLO_API_BOARD_ID')
+
+@pytest.fixture
+def client():
+    file_path = dotenv.find_dotenv('.env.test') 
+    dotenv.load_dotenv(file_path, override=True)
+    
+    test_app = todo_app.app.create_app()
+    
+    with test_app.test_client() as client:
+        yield client
+
+@patch('requests.get')
+def test_index_page(mock_get_requests, client):
+    
+    mock_get_requests.side_effect = mock_trello_request
+    response = client.get('/')
+
+    assert b'New Test' in response.data
+    assert b'sdhjfaj' in response.data
+
+def mock_trello_request(url):
+    trello_board_id = os.getenv('TRELLO_API_BOARD_ID')
+    if url.startswith(f'https://api.trello.com/1/boards/{trello_board_id}/lists'):
+        response = Mock()
+
+        sample_trello_lists_response = [
+            {
+                "id": "601841a7adc438808cfe44b5",
+                "name": "To Do",
+                "closed": False,
+                "pos": 4096,
+                "softLimit": None,
+                "idBoard": "5ff48f05dd6ab805185a793b",
+                "subscribed": False
+            },
+            {
+                "id": "5ff48f05dd6ab805185a793d",
+                "name": "In Progress",
```

Comment:
Here and in your code you use these names, but Trello's default is "To Do", "Doing", "Done". This presumably means you've changed your default so that the e2e test still matches? We should at least mention in the README that that's the case, but ideally we'd also extract out the board names into environment variables